population_trend/__init__.py

```diff
@@ -1,6 +1,6 @@
 """A population growth model package"""
 
-__version__ = "5.9.0"
+__version__ = "6.0.0"
```

Comment:
Por favor sube el número menor y no el mayor.

Hoy revisamos que nadie usa la clase `Plotter_Growth_Rate()`. Esta clase es la que tronaba con las pruebas viejas. Así que no será necesario subir el número mayor.
```suggestion
__version__ = "5.10.0"
```